### DISCUSSION TITLE
"Job was cancelled" when syncing

### DISCUSSION THREAD

**Discussion by rfc2822** [bitfireAT/icsx5#216] [General]
I have upgraded from the previous beta to 2.2-beta.2 and now I always get _Job was cancelled_ when syncing:

![Screenshot_20240226_233214_ICSx](https://github.com/bitfireAT/icsx5/assets/156167/48da7b44-683d-42a8-b7fc-952e7ea856ec)

**Comment by sunkup** [comment]
Fixed in https://github.com/bitfireAT/icsx5/issues/223
